File: easy/array/number of employees who met the target-2798-easy.py
def number_of_employees_who_met_target(hours, target):
    return sum(1 for cur_hours in hours if cur_hours >= target)

result = number_of_employees_who_met_target([0, 1, 2, 3, 4], 2)
print(result)  // Output: 3

File: easy/array/Number-of-employees-who-met-the-target-2798-easy.js
/**
 * @param {number[]} hours
 * @param {number} target
 * @return {number}
 */
var numberOfEmployeesWhoMetTarget = function(hours, target) {
    return hours.reduce((count, curHours) => {
        if (curHours >= target) {
            count++;
        }
        return count;
    }, 0);
};

const result = numberOfEmployeesWhoMetTarget([0,1,2,3,4], 2);
console.log(result); // 3